cypress/e2e/dashboard.cy.js

```diff
@@ -35,13 +35,23 @@ describe('Validate dashboard page', () => {
     cy.get('[data-testid="bannerButton"]').should('be.visible')
   })
 
-  it.skip('Validate 5 Cards (EI,CPP,OAS,SIN,CAL) +2 cards for (CPCD and OAS) and Card titles are Visible', () => {
-    cy.get('[data-testid="myDashboardContent-test"]')
-      .children('[data-cy="cards"]')
-      .should('be.visible')
-      .and('have.length', 7)
-    cy.get('[data-cy="cardtitle"]').should('be.visible').and('have.length', 7)
+  it('Validate 5 Cards (EI,CPP,OAS,SIN,CAL) +2 cards for (CPCD and OAS) and Card titles are Visible', () => {
+    const cardTitles = [
+      'Canadian Dental Care Plan',
+      'Employment Insurance',
+      'Canada Pension Plan',
+      'Old Age Security',
+      'Social Insurance Number',
+      'Canada Apprentice Loan',
+      'Old Age Security',
```

Comment:
I think we could potentially call this something else like `OAS on Curam` instead to differentiate the two different cards but this is fine for now

Reply:
Good Catch, I updated the tests for this issue.